js/src/ui/Theme/theme.js

```diff
@@ -28,11 +28,11 @@ muiTheme.raisedButton.primaryTextColor = 'white';
 muiTheme.snackbar.backgroundColor = 'rgba(255, 30, 30, 0.9)';
 muiTheme.snackbar.textColor = 'rgba(255, 255, 255, 0.9)';
 muiTheme.tabs = lightTheme.tabs;
-muiTheme.tabs.backgroundColor = 'rgb(65, 65, 65)';
+muiTheme.tabs.backgroundColor = 'none'; // 'rgba(65, 65, 65, 0.75)';
```

Comment:
I think we need to get it in mostly as-in and see what bugs us. There will need to be tweaks to make everything fit and look optimal.